plsc/models/resnet.py

```diff
@@ -92,51 +93,79 @@ def conv_bn_layer(self,
             stride=stride,
             padding=pad,
             groups=groups,
-            act=None,
             param_attr=fluid.param_attr.ParamAttr(
```

Comment:
why remove this variable

Reply:
The default value of act for conv2d is none, so i remove it.